test/CMakeLists.txt

```diff
@@ -278,6 +296,20 @@ add_perf_test(test_conv_embed_db
 )
 endif()
 
+add_custom_test(test_conv_for_implicit_gemm_mlir SKIP_UNLESS_ALL
```

Comment:
Would you like to tun these tests on Fiji or on MI100?

Reply:
On Fiji, non MI100. I'm also taking baby steps here as I only turned on the:
- mlir cpp backend
- non-xdlops kernel
- fp32

In the future, I will push in more solvers that also include the following variant:
- mlir bin backend
- xdlops kernel
- fp16

Reply:
Well... don't care about Fiji because we are not developing anything new for it.

However, it seems like this test will run on gfx908. Please fix this. Look for `MIOPEN_TEST_GFX908` to find out how this can be done.

Reply:
Sounds good.

Reply:
I gave it a shot in 42dc9fe. Forgive me if I did not get the idea correctly.

The way I did it is to skip it from running in most scenarios and at the same time use the dedicated jenkins stage to test it. Let me know if you expect a different way.

Reply:
> On Fiji, non MI100. 

Actually the Fwd solver is currently applicable for gfx908. Ditto Bwd and WrW, I guess. So we have two options:
- Disable solver for gfx908 (non-xDLOPs parts), or
- Re-enable tests for gfx908.

Reply:
Let's keep this as is. Acceptable for experimental solvers.